### PR TITLE
New version: kangfenmao.CherryStudio version 0.9.2

### PR DESCRIPTION
**Pull request by spectopo**
Created by [🥟 Dumplings](https://github.com/SpecterShell/Dumplings) in workflow run [#10626](https://github.com/SpecterShell/Dumplings/actions/runs/12535642435).

<details>

<summary>Log</summary>

````
Updated: 0.9.1 → 0.9.2
Submitting WinGet manifests
Creating manifests
Uploading manifests and making commits
Creating a pull request
````

</details>

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/203744)

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]

**Comment by stephengillie** [comment]
URL: https://github.com/kangfenmao/cherry-studio/releases/download/v0.9.2/Cherry-Studio-0.9.2-setup.exe 
Status Code: 200

(Automated message - build 931)

File: manifests/k/kangfenmao/CherryStudio/0.9.2/kangfenmao.CherryStudio.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Created with YamlCreate.ps1 Dumplings Mod
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: kangfenmao.CherryStudio
+PackageVersion: 0.9.2
+PackageLocale: en-US
+Publisher: kangfenmao@qq.com
+PublisherUrl: https://kangfenmao.com/
```
Comment (SpecterShell) [inline manifests/k/kangfenmao/CherryStudio/0.9.2/kangfenmao.CherryStudio.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://cherry-ai.com/
```

File: manifests/k/kangfenmao/CherryStudio/0.9.2/kangfenmao.CherryStudio.locale.zh-CN.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Created with YamlCreate.ps1 Dumplings Mod
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.9.0.schema.json
+
+PackageIdentifier: kangfenmao.CherryStudio
+PackageVersion: 0.9.2
+PackageLocale: zh-CN
+Publisher: kangfenmao@qq.com
+PublisherUrl: https://kangfenmao.com/
```
Comment (SpecterShell) [inline manifests/k/kangfenmao/CherryStudio/0.9.2/kangfenmao.CherryStudio.locale.zh-CN.yaml:8]:
```suggestion
PublisherUrl: https://cherry-ai.com/
```

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**